gm4_double_doors/data/gm4_double_doors/tests/auto_toggle_east_inverted.mcfunction

```diff
@@ -0,0 +1,59 @@
+# @batch east
+# @template gm4_double_doors:tests/valid_door_east_inverted
+# @dummy
```

Comment:
Unless tests clash with each other, you usually don't need to manually batch them, when running them the game will still subdivide each batch into groups of 50 if they go over that, so you can probably leave this in the default batch